.github/workflows/push-pr-lint.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v4
       with:
-        go-version: '1.19'
```

Comment:
the `go-version-file: go.mod` would be nicer, so we don't have to keep bumping this value

Reply:
oh yeah I didn't know about that